fern/docs/pages/user-guide/telemetry.md

```
@@ -0,0 +1,297 @@
+NIXL provides telemetry reader utilities in both C++ and Python for consuming events from the shared memory cyclic buffer. Below are short snippets showing the core reading loop.
+
+<Tip>
+For a complete implementation, see [`examples/python/telemetry_reader.py`](https://github.com/ai-dynamo/nixl/blob/main/examples/python/telemetry_reader.py).
+</Tip>
+
+<CodeBlocks>
+```python title="Python"
+import time
+from nixl_telemetry_reader import SharedRingBuffer
+
+# Open the shared memory telemetry buffer
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)